backend/services/codeExecutionService.js

```
@@ -0,0 +1,237 @@
+            if (typeof solve === 'function') {
+              result = solve(testInput);
+            } else if (typeof twoSum === 'function') {
+              result = twoSum(testInput.nums, testInput.target);
+            } else {
+              throw new Error('No solve or twoSum function found');
+            }
+            
+            JSON.stringify(result);
+          `;
+
+          actualOutput = eval(safeCode);
```

Biome (2.1.2)

[error] eval() exposes to security risks and performance issues.

See the MDN web docs for more details.
Refactor the code so that it doesn't need to call eval().

(lint/security/noGlobalEval)